### ISSUE TITLE
A11y_code.visualstudio.com_VisualStudioCode_Download_Platform Specific_Keyboard: While navigating using keyboard tab key 'Visual studio code in action' list items are not accessible.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [microsoft/vscode-docs#3715]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_Code.visualstudio.com_Benchmark;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#Chrome;#DesktopWeb;#MAS2.1.1;#Win10;#Visual Studio Code;#Keyboard;#CAITestPass_May2020;#FTP;#STP

**Environment Details:**
Application Name: code.visualstudio.com
#URL:[https://code.visualstudio.com](url)
Windows Version: Windows10
Browser version: Microsoft Edge 84.0.488.1 (Official build) dev (64-bit), Google Chrome
 81.0.4044.138

**Repro Steps:**
1. Launch the application Url :[https://code.visualstudio.com](url)
2. Navigate to "Download" button.
3. Hit on any of the platform(Windows/Linux/MacOS) links or any installer type(*64 and *32).
5. Start navigating using keyboard tab key and check weather the 'Visual studio code in action list items are accessible or not.'

**Actual:**
While navigating using keyboard tab key the 'Visual studio code in action list items are not accessible'.

**Expected:**
While navigating using keyboard tab key the 'Visual studio code in action list items should be accessible'.

Attachment  for Reference:
![image](https://user-images.githubusercontent.com/52443697/82981077-b4b27c00-a008-11ea-9c68-c5592eae3a2c.png)

![3715_A11y_code visualstudio com_VisualStudioCode_Download_Platform Specific_Keyboard](https://user-images.githubusercontent.com/52443697/83010548-0ae9e400-a036-11ea-9cff-b466e28a51ce.gif)

**Comment by gregvanl** [comment]
Assigning back to issue author to verify per test pass guidance.

**Comment by Raj-ghuge** [comment]
Verified the bug in environment -  https://code.visualstudio.com/docs/?dv=linux64_deb
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment.

#Closed;
#Regressed:07-19-23;

##Attachments:
[3715_Closed.webm](https://github.com/microsoft/vscode-docs/assets/130441232/8da063d5-69e5-4280-b412-15018f474cd4)
